src/js/game/GameController.js

```diff
@@ -1182,13 +1183,17 @@ class GameController {
       soundEffect = () => this.levelView.audioPlayer.play("fizz");
     }
 
+
     this.levelView.playPlaceBlockInFrontAnimation(player, player.position, player.facing, position, () => {
-      this.levelModel.placeBlockDirection(blockType, placementPlane, player, direction);
-      this.levelView.refreshGroundGroup();
+      if (!this.checkConflict(position, placementPlane)) {
```

Comment:
I think this should happen in the `canPlaceBlockDirection` check in LevelModel?

Reply:
This is probably true. I will move the logic 👍